teams/teams-ps/teams/New-CsTeamsHiddenTemplate.md

```diff
@@ -0,0 +1,70 @@
+---
+external help file: Microsoft.Teams.Policy.Administration.Cmdlets.Core.dll-Help.xml
+Module Name: Microsoft.Teams.Policy.Administration.Cmdlets.Core
+online version: https://learn.microsoft.com/powershell/module/teams/new-csteamshiddentemplate
+title: New-CsTeamsHiddenTemplate
+author: yishuaihuang
+ms.author: yishuaihuang
+ms.reviewer: 
+manager: weiliu2
+schema: 2.0.0
+---
+
+# New-CsTeamsHiddenTemplate
+
+## SYNOPSIS
+This cmdlet is used to create a `HiddenTemplate` object for use with the [New-CsTeamsTemplatePermissionPolicy](New-CsTeamsTemplatePermissionPolicy.md) and [Set-CsTeamsTemplatePermissionPolicy](Set-CsTeamsTemplatePermissionPolicy.md) cmdlets.
+
+## SYNTAX
+
+```
+New-CsTeamsHiddenTemplate -Id <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates an object that can be supplied as `HiddenTemplate` to the [New-CsTeamsTemplatePermissionPolicy](New-CsTeamsTemplatePermissionPolicy.md) and [Set-CsTeamsTemplatePermissionPolicy](Set-CsTeamsTemplatePermissionPolicy.md) cmdlets.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $manageProjectTemplate = New-CsTeamsHiddenTemplate -Id com.microsoft.teams.template.ManageAProject
+```
+
+Creates a new hidden teams template object. For more examples of how this can be used, see the examples for [New-CsTeamsTemplatePermissionPolicy](New-CsTeamsTemplatePermissionPolicy.md).
```

Comment:
```suggestion
Creates a new hidden Teams template object. For more examples of how this can be used, see the examples for [New-CsTeamsTemplatePermissionPolicy](New-CsTeamsTemplatePermissionPolicy.md).
```